src/components/DivElement.js

```diff
@@ -1,6 +1,14 @@
 const HTMLElement = require('./HTMLElement');
 
-// Define class here
+class DivElement extends HTMLElement {
+  constructor(content) {
+    super();
```

Comment:
You can pass the parameters into super to use the parents constructor to set them. ie:
super('div', content);